lib/elixir/lib/string.ex

```diff
@@ -1812,4 +1812,104 @@ defmodule String do
       {comm + 1, trans, current}
     end
   end
+
+  @doc """
+  Returns a keyword list that represents an edit graph.
+
+  The algorithm is outlined in the "An O(ND) Difference Algorithm and Its Variations" paper.
```

Comment:
I thought about that, but doesn't seem there is an official/reliable link exists.